src/client/components/builder/logic/CalculatorBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Flex, FlexProps, Text } from '@chakra-ui/layout'
+import React, { FC } from 'react'
+
+interface CalculatorBarProps extends Omit<FlexProps, 'onClick'> {
+  onClick: (character: string) => void
+}
+
+export const CalculatorBar: FC<CalculatorBarProps> = ({
+  onClick,
+  ...props
+}) => {
+  const operands = '@+-×÷()^%>≥<≤=≠'.split('')
```

Comment:
We should create a map of operator to mappedOperator so that we won't run the risk of messing up the indices when we add/remove more operators in the future, e.g.
```
const operatorMap = { 
  "@": "@",
  "=": "==",
   ...
} 
```

P.S. it should be **operator** not **operand** - **operand** refers to the object of the operation, i.e. in `3 + 6`, `3` and `6` are the **operands** while `+` is the **operator**